src/components-styled/content-header/index.tsx

```diff
@@ -0,0 +1,115 @@
+import css from '@styled-system/css';
+import Link from 'next/link';
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+import {
+  Metadata,
+  MetadataProps,
+} from '~/components-styled/content-header/metadata';
+import { HeadingWithIcon } from '~/components-styled/heading-with-icon';
+import { Heading, Text } from '~/components-styled/typography';
+import { Box, BoxProps } from '../base';
+
+const HeaderBox = styled.header<BoxProps>({
+  '&[id]': {
+    marginLeft: '-100vw',
+    paddingLeft: '100vw',
+  },
+});
+
+interface HeaderProps {
+  id?: string;
+  hasCategory: boolean;
+  hasIcon: boolean;
+  children: ReactNode;
+}
+
+const Header = (props: HeaderProps) => {
+  const { hasCategory, hasIcon, children, id } = props;
+  return (
+    <HeaderBox
+      id={id}
+      marginTop={hasCategory ? undefined : 4}
+      marginLeft={hasIcon ? undefined : 5}
+    >
+      {children}
+    </HeaderBox>
+  );
+};
+
+const CategoryText = styled(Text)(
+  css({
+    fontSize: 3,
+    fontWeight: 'bold',
+    color: 'category',
+    margin: 0,
+    marginBottom: 1,
+    marginLeft: 5,
+  })
+);
+
+const BodyBox = styled(Box)(
+  css({
+    display: [null, null, null, 'flex'],
+    marginLeft: [null, null, null, 5],
```

Comment:
Is it possible to use the object syntax?

```suggestion
    display: { lg: 'flex' },
    marginLeft: { lg: 5 },
```

Reply:
Nope, unfortunately not, I get compile errors if I use the object syntax.

Reply:
I remember running into that as well .. we need to dive into that some day